src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetUpdateMethod.cs

```diff
@@ -251,7 +251,7 @@ public override void ExecuteCmdlet()
 
         [Parameter(
             Mandatory = false)]
-        [ValidateNotNullOrEmpty]
+        [AllowEmptyString]
```

Comment:
Can you help me understand why this class doesn't need to set ProximityPlacementGroupId to null like others?

Reply:
The ask was allow an empty string value for ProximityPlacementGroupId to support the scenario of removing ProximityPlacementGroupId from VM.   In REST request, setting the ID to null or setting the ID to empty string have the same effect, but ProximityPlacementGroup should not be null.  (If ProximityPlacementGroup is null, then no change of ProximityPlacementGroupId).  So, I think we can accept null value here (for ID), but it may be better that a user to give an empty string explicitly to avoid unintended removal.

Reply:
understood, but in other two cmdlets you changed, after [AllowEmptyString], there are if conditions to set
"avSetParams.ProximityPlacementGroup.Id = null;"
and 
"parameters.ProximityPlacementGroup.Id = null;"

I'm just wondering why this cmdlet doesn't have such thing.

Reply:
Thanks.  I updated the PR by adding the same logic to VMSS.